ubuntudesign/www.ubuntu.com
# updating options/required labels

## Done

* removed * for required fields and added '(optional)' for non-required fields

## QA

1. go to:
* /cloud/contact-us
* /cloud/openstack/managed-cloud/contact-us
* /cloud/openstack/training/classroom-contact-us
* /cloud/openstack/training/contact-us
* /cloud/openstack/training/onsite-contact-us
* /cloud/storage/contact-us
* /desktop/contact-us
* /desktop/tenders-contact-us
* /internet-of-things/contact-us
* /internet-of-things/digital-signage/contact-us
* /legal/terms-and-policies/contact-us
* /management/contact-us
* /server/contact-us
* /tablet/contact-us

and see that there are no * and very rarely you will see a (optional) around fields... like Country and Job roles

## Issue / Card

[List of links to Github issues/bugs and cards if needed]

Fixes #649
